### PR TITLE
Add missing BinaryTargets fixture xcodeproj

### PR DESCRIPTION
**Pull request by hartbit**

### CODE REVIEW

**Comment by hartbit** [comment]
@swift-ci smoke test

File: IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+			productType = "com.apple.product-type.framework";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		63A1B3DB23FB4B26001B7732 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastUpgradeCheck = 1140;
+				ORGANIZATIONNAME = "David Hart";
```
Comment (aciidb0mb3r) [inline IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj]: can you use swiftpm-fixture or something generic here?

File: IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+		};
+		63A1B3ED23FB4B26001B7732 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				BUILD_LIBRARY_FOR_DISTRIBUTION = YES;
+				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_STYLE = Automatic;
+				COMBINE_HIDPI_IMAGES = YES;
+				DEFINES_MODULE = YES;
+				DEVELOPMENT_TEAM = AM5AA6A5KV;
```
Comment (aciidb0mb3r) [inline IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj]: This should be empty

File: IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				INFOPLIST_FILE = SwiftFramework/Info.plist;
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/../Frameworks",
+					"@loader_path/Frameworks",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = com.hartbit.SwiftFramework;
```
Comment (aciidb0mb3r) [inline IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj]: How about org.swift.swiftpm.fixture.SwiftFramework?

Comment (hartbit) [reply IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj]: Fixed!

**Comment by hartbit** [comment]
@swift-ci smoke test

File: IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj (hunk 2 above)
Comment (hartbit) [reply IntegrationTests/Fixtures/BinaryTargets/Inputs/SwiftFramework/SwiftFramework.xcodeproj/project.pbxproj]: Done

**Comment by hartbit** [comment]
@swift-ci smoke test

**Comment by hartbit** [comment]
@swift-ci smoke test macOS

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test macOS

**Pull request merged**